terraform/waf/modules/test/rule_group.tf

```diff
@@ -0,0 +1,32 @@
+resource "aws_waf_rule" "wafrule_2" {
```

Comment:
[non-blocking] Should we follow the naming like other PRs and use `${var.prefix}-`? e.g. instead of `name = "${var.prefix}wafrule2"` use `name = "${var.prefix}-wafrule2"`

If this is due to a constraint on the name (it can't have `-` in it), please ignore

Reply:
Because this specific resource does not support non alphanumeric chars